pkcs11/src/new/types/mechanism/mod.rs

```diff
@@ -339,356 +363,397 @@ pub enum Mechanism {
     RsaPkcsTpm_1_1,
     RsaPkcsOaepTpm_1_1,
 }
+*/
+
+impl From<MechanismType> for CK_MECHANISM_TYPE {
+    fn from(mechanism_type: MechanismType) -> Self {
+        match mechanism_type {
+            RSA_PKCS_KEY_PAIR_GEN => CKM_RSA_PKCS_KEY_PAIR_GEN,
+            RSA_PKCS => CKM_RSA_PKCS,
+            _ => panic!("Malformed MechanismType was not expected."),
+            /*
+            MechanismType::RsaPkcs => CKM_RSA_PKCS,
```

Comment:
Shall I delete the ones that we do not need in Parsec?

Reply:
I'd say so, yeah